sumomongodbatlascollector/api.py

```diff
@@ -57,7 +66,6 @@ def fetch(self):
         state = None
         payload = []
         try:
-
             fetch_success, content = ClientMixin.make_request(url, method="get", logger=self.log, TIMEOUT=self.collection_config['TIMEOUT'], MAX_RETRY=self.collection_config['MAX_RETRY'], BACKOFF_FACTOR=self.collection_config['BACKOFF_FACTOR'], **kwargs)
```

Comment:
can we add one more logline here as well with  
LogType: {log_type} kwargs: {kwargs} status: {fetch_success} url: {url}
execution time: {end_time - start_time:.2f} seconds current_process_memory_usage: {self._get_current_process_memory_usage()}

Reply:
Also at line 77 and 79 also lets add kwargs url execution time and current_process_memory_usage

Reply:
Moved to the context manager based logging